gdal.cmake

```diff
@@ -256,6 +256,14 @@ function (gdal_add_private_link_libraries)
 endfunction (gdal_add_private_link_libraries)
 
 add_library(${GDAL_LIB_TARGET_NAME} gcore/gdal.h)
+
+# If a shared lib renaming has been set in ConfigUser.cmake
+if (GDAL_SHAREDLIB_OUTPUT_NAME)
```

Comment:
it should be documented in build_hints.rst

Reply:
It is but it refers ``cmake/ConfigUser.cmake`` so that needs to be changed after first point is solved.